docs/guides/README.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Guides
+children:
+  - anchoralarm/anchoralarm.md
+  - datalogging/datalogging.md
+  - navdataserver/navdataserver.md
+---
+
+<!-- TODO -->
```

Comment:
It would be good to have a short summary here rather than nothing.

Reply:
Thanks, good catch. Typedoc wouldn't let me group these without creating an intermediary page. I'll come up with something here.